### PR TITLE
Update config split documentation

### PR DESCRIPTION
**Pull request by danepowell**
Starting to add documentation specifically targeted at TAs and developers.

### CODE REVIEW

File: readme/configuration-management.md [hunk 1]
```diff
@@ -70,14 +70,39 @@ We need to find a better way of preventing this than manually monitoring module
 
 ## Configuration Split workflow
 
-BLT recommends using the Config Split module to manage configuration on most projects. For a description of how to use Config Split, see this excellent [blog post by Jeff Geerling](https://www.jeffgeerling.com/blog/2017/adding-configuration-split-drupal-site-using-blt-and-acquia-cloud).
+### Overview
+
+BLT recommends using the Config Split module to manage configuration on most projects. For an overview of how to use Config Split, see this excellent [blog post by Jeff Geerling](https://www.jeffgeerling.com/blog/2017/adding-configuration-split-drupal-site-using-blt-and-acquia-cloud).
```
Comment (grasmash) [inline readme/configuration-management.md]: For backwards compatibility, BLT currently defaults to using features. We could instead default to config split and introduce an update hook to set the cm.strategy to "features" for projects created before 8.7.*. Thoughts?

File: readme/configuration-management.md [hunk 2]
```diff
@@ -70,14 +70,39 @@ We need to find a better way of preventing this than manually monitoring module
...
 
 TODO: Update this documentation once Config Split's greylist functionality has been better documented and tested.
 
 The second limitation is that it's difficult to define configuration that varies between sites in a multisite installation. Multisite installations that require highly customized bundles of configuration per-site might be better suited by a Features-based workflow.
 
+### Setting up Config Split
+
+This section will guide you through initially setting up a Config Split workflow on your BLT based project.
+
+1. Ensure that you are using Drush version 8.1.10 or higher.
```
Comment (grasmash) [inline readme/configuration-management.md]: This is already required by BLT in 8.7.*.

File: readme/configuration-management.md [hunk 3]
```diff
@@ -70,14 +70,39 @@ We need to find a better way of preventing this than manually monitoring module
...
+2. Add Config Split version 8.x-1.0-beta4 or higher to your codebase: `composer require drupal/config_split`
```
Comment (grasmash) [inline readme/configuration-management.md]: This is already included in composer.required.json.

File: readme/configuration-management.md [hunk 4]
```diff
@@ -70,14 +70,39 @@ We need to find a better way of preventing this than manually monitoring module
...
+3. TODO: finish this section once the related functionality is stabilized in BLT core.
```
Comment (grasmash) [inline readme/configuration-management.md]: We also need to indicate that `cm.strategy` must be set to `config-split` in project.yml. We may make this the default.

File: readme/configuration-management.md [hunk 5]
```diff
@@ -70,14 +70,39 @@ We need to find a better way of preventing this than manually monitoring module
...
+4. Review the updated configuration in `config/default` using `git diff`.  If you are satisfied with the changes, commit them and open a pull request.
+
+Note that when you run `drush cex`, if the project has been configured correctly, some configuration that's specific to the development environment should automatically be excluded. Similarly, some configuration that's intended to be "unlocked" in production might also be excluded (such as webforms). If you need to customize this behavior, you can modify the blacklists or greylists by following the steps in the section above.
+
 ## Features-based workflow
 
 Features allows you to bundle related configuration files (such as a content type and its fields) into individual feature modules. Drupal treats features just like normal modules, but Features and its dependencies add some special sauce that allow features to not only provide default configuration (like normal modules), but to also update (track and import) changes to this configuration.
```
Comment (grasmash) [inline readme/configuration-management.md]: We need to indicate that `cm.strategy` must be set to `features` in project.yml.

File: readme/configuration-management.md (hunk 1 above)
Comment (danepowell) [inline readme/configuration-management.md]: Yeah I definitely think we should default to config_split. An update hook (or just a line in the release notes) for existing features-based projects sounds good to me.

File: readme/configuration-management.md (hunk 5 above)
Comment (danepowell) [inline readme/configuration-management.md]: done

**Pull request merged**